fili-core/src/main/java/com/yahoo/bard/webservice/data/config/table/MetricUnionCompositeTableDefinition.java

```diff
@@ -93,7 +99,12 @@ public PhysicalTable build(ResourceDictionaries dictionaries, DataSourceMetadata
     private Set<PhysicalTable> getPhysicalTables(ResourceDictionaries resourceDictionaries) {
         PhysicalTableDictionary physicalTableDictionary = resourceDictionaries.getPhysicalDictionary();
         return dependentTableNames.stream()
-                .map(name -> physicalTableDictionary.getOrDefault(name, null))
+                .peek(name -> {
```

Comment:
Also note that the log message will indicate that it's the `MetricUnionCompositeTableDefinition` class that's emitting the log, so including that in the message isn't strictly needed (though it doesn't hurt)